srcpkgs/libinput-gestures/INSTALL.msg

```diff
@@ -0,0 +1,11 @@
+
+	Get started with libinput-gestures:
+
+		$ libinput-gestures-setup autostart
+		$ libinput-gestures-setup start
+
+	The default configuration is stored at /etc/libinput-gestures.conf.
+	Edit it by copying it to ~/.config/libinput-gestures.conf
```

Comment:
libinput gestures uses the XDG_CONFIG_HOME variable which can be different for users (i.e i use $HOME/etc) 

overall i don't believe this has much use since people explicitly installing this already know how activate it